src/main/java/com/treasuredata/client/model/TDDatabase.java

```diff
@@ -30,6 +30,8 @@ public class TDDatabase
     private final String id;
     private final String name;
     private final long count;
+    private final long userId;
```

Comment:
In td-client-java, `user_id` is handled as an integer value as follows:
https://github.com/treasure-data/td-client-java/blob/75366cdfe7eba1018aeb4d253ba045fb905cebc8/src/main/java/com/treasuredata/client/model/TDApiKey.java#L16

It's good to follow the existing data type for `user_id` as much as possible.